### PR TITLE
Update python 3.6 to 3.6.0b2

### PR DESCRIPTION
**Pull request by IlyaSkriblovsky**

### CODE REVIEW

File: library/python [hunk 1]
```diff
@@ -90,23 +90,23 @@ GitCommit: 855b85c8309e925814dfa97d61310080dcd08db6
...
-Tags: 3.6.0b1-onbuild, 3.6-onbuild
-GitCommit: 635ea5d58b53d165f7bedae90f8933c720a58150
+Tags: 3.6.0b2-onbuild, 3.6-onbuild
+GitCommit: aa533a4879c13944d62a5029929ffae78075d212
 Directory: 3.6/onbuild
 
-Tags: 3.6.0a4-windowsservercore, 3.6-windowsservercore
-GitCommit: 855b85c8309e925814dfa97d61310080dcd08db6
+Tags: 3.6.0b2-windowsservercore, 3.6-windowsservercore
+GitCommit: aa533a4879c13944d62a5029929ffae78075d212
```
Comment (tianon) [inline library/python]: We actually need to revert the `windowsservercore` update back to 3.6.0a4 thanks to https://github.com/docker-library/python/issues/146 :cry:

**Comment by IlyaSkriblovsky** [comment]
Ash, sorry I thought it was fixed in 3.6.0b2. I will revert this one

File: library/python (hunk 1 above)
Comment (tianon) [inline library/python]: Don't bother updating!  Turns out it was a Windows issue (https://github.com/docker-library/python/issues/146#issuecomment-253315389):

> Soo....  I just tested Windows Server 2016 (RTM) which was officially made available today, and this is fixed there.  Looks like it was a Windows issue, not a Python issue.

**Comment by IlyaSkriblovsky** [comment]
Reverted `windowsservercore` back

**Comment by IlyaSkriblovsky** [comment]
aah, saw your post too late. Should I change `windowsservercore` to b2 again?

**Comment by tianon** [comment]
@IlyaSkriblovsky :disappointed: I was just a second too late commenting, looks like

If you don't mind, switching it back would be great! :+1:  :heart:

**Comment by IlyaSkriblovsky** [comment]
@tianon done!

**Comment by tianon** [comment]
Thanks! :+1: :heart:

```diff
diff --git a/python_3.6-alpine/Dockerfile b/python_3.6-alpine/Dockerfile
index faa66a9..99e37bc 100644
--- a/python_3.6-alpine/Dockerfile
+++ b/python_3.6-alpine/Dockerfile
@@ -12,7 +12,7 @@ ENV LANG C.UTF-8
 RUN apk add --no-cache ca-certificates
 
 ENV GPG_KEY 0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D
-ENV PYTHON_VERSION 3.6.0b1
+ENV PYTHON_VERSION 3.6.0b2
 
 # if this is called "PIP_VERSION", pip explodes with "ValueError: invalid truth value '<VERSION>'"
 ENV PYTHON_PIP_VERSION 8.1.2
diff --git a/python_3.6-slim/Dockerfile b/python_3.6-slim/Dockerfile
index 89f6f6d..faf5298 100644
--- a/python_3.6-slim/Dockerfile
+++ b/python_3.6-slim/Dockerfile
@@ -15,7 +15,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/*
 
 ENV GPG_KEY 0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D
-ENV PYTHON_VERSION 3.6.0b1
+ENV PYTHON_VERSION 3.6.0b2
 
 # if this is called "PIP_VERSION", pip explodes with "ValueError: invalid truth value '<VERSION>'"
 ENV PYTHON_PIP_VERSION 8.1.2
diff --git a/python_3.6-windowsservercore/Dockerfile b/python_3.6-windowsservercore/Dockerfile
index b5fa44e..771a30a 100644
--- a/python_3.6-windowsservercore/Dockerfile
+++ b/python_3.6-windowsservercore/Dockerfile
@@ -2,7 +2,7 @@ FROM microsoft/windowsservercore
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
 
-ENV PYTHON_VERSION 3.6.0a4
+ENV PYTHON_VERSION 3.6.0b2
 ENV PYTHON_RELEASE 3.6.0
 
 # if this is called "PIP_VERSION", pip explodes with "ValueError: invalid truth value '<VERSION>'"
diff --git a/python_3.6/Dockerfile b/python_3.6/Dockerfile
index b7760f7..bc0af85 100644
--- a/python_3.6/Dockerfile
+++ b/python_3.6/Dockerfile
@@ -14,7 +14,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/*
 
 ENV GPG_KEY 0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D
-ENV PYTHON_VERSION 3.6.0b1
+ENV PYTHON_VERSION 3.6.0b2
 
 # if this is called "PIP_VERSION", pip explodes with "ValueError: invalid truth value '<VERSION>'"
 ENV PYTHON_PIP_VERSION 8.1.2
```

**Comment by tianon** [comment]
Windows build test:
```console
Building bashbrew/cache:0b8bec13bd007e9283163dd9483d0418652ce7f8700a470d162fb72c027bb7b8 (python:3.6.0b2-windowsservercore)
Tagging python:3.6.0b2-windowsservercore
Tagging python:3.6-windowsservercore
```

**Comment by tianon** [comment]
LGTM

Build test of #2242; d1a35fa (`python`):

```console

$ bashbrew build python:2.7.12
Using bashbrew/cache:7609b95f0fb1eb903779427343c6e721655b4e8c5f00ce8e1c19dcd0dc714333 (python:2.7.12)
Tagging python:2.7.12
Tagging python:2.7
Tagging python:2

$ test/run.sh python:2.7.12
testing python:2.7.12
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:2.7.12-slim
Building bashbrew/cache:47f6c33d3d097be313edfff684dd0e65da3c3dc028f896dc25a22ea40d99b49e (python:2.7.12-slim)
Tagging python:2.7.12-slim
Tagging python:2.7-slim
Tagging python:2-slim

$ test/run.sh python:2.7.12-slim
testing python:2.7.12-slim
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:2.7.12-alpine
Building bashbrew/cache:839c744e57aa9aa8a979714a3c2630422723100000f7539ac155f1fb3a018cf4 (python:2.7.12-alpine)
Tagging python:2.7.12-alpine
Tagging python:2.7-alpine
Tagging python:2-alpine

$ test/run.sh python:2.7.12-alpine
testing python:2.7.12-alpine
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:2.7.12-wheezy
Using bashbrew/cache:9a3c7d74ffa89c4a8565b4438fdfd93b35f9fe30c75c5e7befe9d5ab996cbe45 (python:2.7.12-wheezy)
Tagging python:2.7.12-wheezy
Tagging python:2.7-wheezy
Tagging python:2-wheezy

$ test/run.sh python:2.7.12-wheezy
testing python:2.7.12-wheezy
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:2.7.12-onbuild
Using bashbrew/cache:c5b8e28aeef3acdbd2ab475f55c6e71e24ac7c4473f33b1278a2d7d1ffb394a2 (python:2.7.12-onbuild)
Tagging python:2.7.12-onbuild
Tagging python:2.7-onbuild
Tagging python:2-onbuild

$ test/run.sh python:2.7.12-onbuild
testing python:2.7.12-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build python:2.7.12-windowsservercore
skipping "python:2.7.12-windowsservercore" (due to unsatisfactory constraints ["windowsservercore"])

$ test/run.sh python:2.7.12-windowsservercore
testing python:2.7.12-windowsservercore
	image does not exist!


$ bashbrew build python:3.3.6
Using bashbrew/cache:a6379979afeed93a5c39f979af815e11affa7b31124085a9accdaa163f1578c5 (python:3.3.6)
Tagging python:3.3.6
Tagging python:3.3

$ test/run.sh python:3.3.6
testing python:3.3.6
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.3.6-slim
Building bashbrew/cache:87a07207bca915ce6e088f7eeeaf4eb54077a3228cae216c2296dbc4a14f277f (python:3.3.6-slim)
Tagging python:3.3.6-slim
Tagging python:3.3-slim

$ test/run.sh python:3.3.6-slim
testing python:3.3.6-slim
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.3.6-alpine
Building bashbrew/cache:2a2b7ccd213f0e4e3c6ae87f291622c04ec58a116b2a09b5c2f9677ae6d9ab89 (python:3.3.6-alpine)
Tagging python:3.3.6-alpine
Tagging python:3.3-alpine

$ test/run.sh python:3.3.6-alpine
testing python:3.3.6-alpine
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.3.6-wheezy
Using bashbrew/cache:e2c755e9ba1adfb4f7041ecb249eb8f1313cc3f891bf81595fe3cd65dae51509 (python:3.3.6-wheezy)
Tagging python:3.3.6-wheezy
Tagging python:3.3-wheezy

$ test/run.sh python:3.3.6-wheezy
testing python:3.3.6-wheezy
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.3.6-onbuild
Using bashbrew/cache:ff7b11d9ea8d48ce0d8ddd441ee8dc94e953dcff763372a510fa858a28940417 (python:3.3.6-onbuild)
Tagging python:3.3.6-onbuild
Tagging python:3.3-onbuild

$ test/run.sh python:3.3.6-onbuild
testing python:3.3.6-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build python:3.4.5
Using bashbrew/cache:134f43d4963b0aff61c0ea622a3b31b819768dab2366d299a59070e502e42804 (python:3.4.5)
Tagging python:3.4.5
Tagging python:3.4

$ test/run.sh python:3.4.5
testing python:3.4.5
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.4.5-slim
Building bashbrew/cache:033cfee68ef77040a4a519cb4ee004d78e4b77dd160d96e68ef73f724a0b3459 (python:3.4.5-slim)
Tagging python:3.4.5-slim
Tagging python:3.4-slim

$ test/run.sh python:3.4.5-slim
testing python:3.4.5-slim
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.4.5-alpine
Building bashbrew/cache:669471df1454cc7aff2566be6f537254dfc5d4903f34ec09c0fde3d9de8ff6d6 (python:3.4.5-alpine)
Tagging python:3.4.5-alpine
Tagging python:3.4-alpine

$ test/run.sh python:3.4.5-alpine
testing python:3.4.5-alpine
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.4.5-wheezy
Using bashbrew/cache:ec10d91582faf6ddc7ff4f1203dbef36a4d23b8af51ec0590922277c6031fc71 (python:3.4.5-wheezy)
Tagging python:3.4.5-wheezy
Tagging python:3.4-wheezy

$ test/run.sh python:3.4.5-wheezy
testing python:3.4.5-wheezy
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.4.5-onbuild
Using bashbrew/cache:46a6ee30898b5c2637872e3f504c348ef3fe8523a2f9e202386b2ba8d902870e (python:3.4.5-onbuild)
Tagging python:3.4.5-onbuild
Tagging python:3.4-onbuild

$ test/run.sh python:3.4.5-onbuild
testing python:3.4.5-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build python:3.5.2
Using bashbrew/cache:204e7019cf24afe848fe5ecf2632d6f98297bc3945cca42c78260aee7a0f3a44 (python:3.5.2)
Tagging python:3.5.2
Tagging python:3.5
Tagging python:3
Tagging python:latest

$ test/run.sh python:3.5.2
testing python:3.5.2
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.5.2-slim
Building bashbrew/cache:f6d48868d33f10b5eea3618d41e99e0d8702978f27a5ff17c4f0e187418ece6a (python:3.5.2-slim)
Tagging python:3.5.2-slim
Tagging python:3.5-slim
Tagging python:3-slim
Tagging python:slim

$ test/run.sh python:3.5.2-slim
testing python:3.5.2-slim
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.5.2-alpine
Building bashbrew/cache:4d971ee199bf3fedc13267d9e6401c2a31ed731af1243f7ecfdccf0d87e8c2d2 (python:3.5.2-alpine)
Tagging python:3.5.2-alpine
Tagging python:3.5-alpine
Tagging python:3-alpine
Tagging python:alpine

$ test/run.sh python:3.5.2-alpine
testing python:3.5.2-alpine
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.5.2-onbuild
Using bashbrew/cache:e9018e6456998ae95e385377cbed658981004f12c1d049c07b7ec90b7d237d6a (python:3.5.2-onbuild)
Tagging python:3.5.2-onbuild
Tagging python:3.5-onbuild
Tagging python:3-onbuild
Tagging python:onbuild

$ test/run.sh python:3.5.2-onbuild
testing python:3.5.2-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build python:3.5.2-windowsservercore
skipping "python:3.5.2-windowsservercore" (due to unsatisfactory constraints ["windowsservercore"])

$ test/run.sh python:3.5.2-windowsservercore
testing python:3.5.2-windowsservercore
	image does not exist!


$ bashbrew build python:3.6.0b2
Building bashbrew/cache:f82e071e9e98b9c6f52a1f03e9f6d8a231500891c8b30f201368e5cdc105a878 (python:3.6.0b2)
Tagging python:3.6.0b2
Tagging python:3.6

$ test/run.sh python:3.6.0b2
testing python:3.6.0b2
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...skipping Hy test -- no workie on Python 3.6+
passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.6.0b2-slim
Building bashbrew/cache:3f5e48c489a907985e3c300daa5a1a1d25a70862031b19d81a967ff0363a4fb7 (python:3.6.0b2-slim)
Tagging python:3.6.0b2-slim
Tagging python:3.6-slim

$ test/run.sh python:3.6.0b2-slim
testing python:3.6.0b2-slim
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...skipping Hy test -- no workie on Python 3.6+
passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.6.0b2-alpine
Building bashbrew/cache:07c84b535ef12954220dbae75e73efd9e160e38663e14e6c9a9ca412b0921561 (python:3.6.0b2-alpine)
Tagging python:3.6.0b2-alpine
Tagging python:3.6-alpine

$ test/run.sh python:3.6.0b2-alpine
testing python:3.6.0b2-alpine
	'utc' [1/8]...passed
	'cve-2014--shellshock' [2/8]...passed
	'no-hard-coded-passwords' [3/8]...passed
	'override-cmd' [4/8]...passed
	'python-hy' [5/8]...skipping Hy test -- no workie on Python 3.6+
passed
	'python-imports' [6/8]...passed
	'python-pip-requests-ssl' [7/8]...passed
	'python-sqlite3' [8/8]...passed


$ bashbrew build python:3.6.0b2-onbuild
Building bashbrew/cache:0110a68815217a82ed86fc7b9501114dc2678a50ff5438a3f1d9e8d1e37b65e1 (python:3.6.0b2-onbuild)
Tagging python:3.6.0b2-onbuild
Tagging python:3.6-onbuild

$ test/run.sh python:3.6.0b2-onbuild
testing python:3.6.0b2-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build python:3.6.0b2-windowsservercore
skipping "python:3.6.0b2-windowsservercore" (due to unsatisfactory constraints ["windowsservercore"])

$ test/run.sh python:3.6.0b2-windowsservercore
testing python:3.6.0b2-windowsservercore
	image does not exist!

```

The following images failed at least one test: python:2.7.12-windowsservercore python:3.5.2-windowsservercore python:3.6.0b2-windowsservercore

**Pull request merged**